user_templates/mcu_defaults/ercf/Tircown_ERCF_easy_brd.cfg

```diff
@@ -25,4 +25,3 @@ aliases:
     ERCF_SERVO=MCU_SERVO ,
 
     ERCF_ENCODER=MCU_ENCODER ,
-
```

Comment:
Can you put back this second free new line at the end? It was there on purpose to allow better separation when injecting the templates from the install script